source/client/factories_impl.cc

```diff
@@ -31,18 +31,28 @@ BenchmarkClientPtr BenchmarkClientFactoryImpl::create(
     Envoy::Api::Api& api, Envoy::Event::Dispatcher& dispatcher, Envoy::Stats::Scope& scope,
     Envoy::Upstream::ClusterManagerPtr& cluster_manager,
     Envoy::Tracing::HttpTracerSharedPtr& http_tracer, absl::string_view cluster_name,
-    RequestSource& request_generator) const {
+    const int sink_stat_prefix, RequestSource& request_generator) const {
```

Comment:
We probably don't need to mark an int argument const, since int being a basic type is immutable (passed by value).

Reply:
Done.